send2cgeo.user.js

```diff
@@ -389,10 +389,9 @@ function s2cgGCMain() {
     if (document.location.href.match(/\.com\/(seek\/cache_details\.aspx|geocache\/)/)) {
         var GCCode = $("#ctl00_ContentBody_CoordInfoLinkControl1_uxCoordInfoCode").html();
 
-        var html2 = '<dt class="label">'
-            + '<a id="ctl00_ContentBody_lnkGpxDownload" onclick="window.s2geo(\'' + GCCode
-            + '\'); return false;"><span id="ctl00_ContentBody_uxDownloadLabel">Send to c:geo</span></a>'
-            + '</dt>'
+        var html2 = '<a href="javascript:void(0);" onclick="window.s2geo(\'' + GCCode + '\'); return false;">'
+			+ '<img src="https://send2.cgeo.org/send2cgeo.png" title="Send to c:geo" height="16px" />'
+			+ '<span>Send to c:geo</span></a>';
```

Comment:
You can wrap the html in <dt> ... </dt>, but without class definition.
Then we have the same text formation, but no download icon.